backend/inspirehep/indexer/cli.py

```diff
@@ -72,7 +68,7 @@ def _prepare_logdir(log_path):
         makedirs(path.dirname(log_path))
 
 
-@indexer.command("reindex")
+@index.command("reindex")
```

Comment:
maybe add a help to indicates that this is the inspire one

Reply:
This is why I added this (Inspire) Thing so it will be easy distinguishable that it's inspire one ;) - First sentence from doc-string is displayed as a description for the command.
```bash
[pazembrz@laptop backend]$ poetry run inspirehep index --help
Usage: inspirehep index [OPTIONS] COMMAND [ARGS]...

  Manage search indices.

Options:
  --help  Show this message and exit.

Commands:
  check    Check Elasticsearch version.
  create   Create a new index.
  delete   Delete index by its name.
  destroy  Destroy all indexes.
  init     Initialize registered aliases and mappings.
  list     List indices.
  put      Index input data.
  queue    Manage indexing queue.
  reindex  (Inspire) Reindex records in ElasticSearch.
  remap    (Inspire) Remaps specified indexes.
  run      Run bulk record indexing.
```

Reply:
@drjova